src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0;i < 3;i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes
Bad example:
```
public class Main {
  public static void main(String[] args) {
      String[] emails = {"myEmail@gmail.com", "not@Valid@.g.com"};
      for(String email : emails) {
          if (new EmailValidator().isValid(email)) {
              System.out.println("Email " + email + " is valid");
          }
      }
  }
```
}
Improved example:
```
public class Main {
  public static void main(String[] args) {
      String[] emails = {"myEmail@gmail.com", "not@Valid@.g.com"};
      EmailValidator emailValidator = new EmailValidator();
      for(String email : emails) {
          if (emailValidator.isValid(email)) {
              System.out.println("Email " + email + " is valid");
          }
      }
  }
}

```